homeassistant/components/modbus/switch.py

```diff
@@ -120,7 +120,6 @@ async def async_turn_on(self, **kwargs):
             self._available = False
             self.async_write_ha_state()
         else:
-            self._available = True
```

Comment:
It looks like the branch isn't completely rebased on latest dev branch. There should just be two new commits in the branch I think, and this file shouldn't be touched.

Reply:
Fixed (it is not enough to do "git rebase dev", it helps a lot if you start by doing "git pull" on the dev branch).